app/src/main/java/org/p2p/wallet/swap/ui/jupiter/tokens/SwapTokensFragment.kt

```diff
@@ -63,11 +65,17 @@ class SwapTokensFragment :
         binding.toolbar.setNavigationOnClickListener { popBackStack() }
         inflateSearchMenu(binding.toolbar)
 
-        binding.recyclerViewTokens.attachAdapter(adapter)
-        when (tokenToChange) {
-            SwapTokensListMode.TOKEN_A -> binding.recyclerViewTokens.addItemDecoration(SwapTokenAItemDecoration())
-            SwapTokensListMode.TOKEN_B -> {
-                // todo
+        with(binding.recyclerViewTokens) {
+            attachAdapter(this@SwapTokensFragment.adapter)
```

Comment:
Where is layout manager? 
Please set it in code, it's not obvious if it's in xml.
If we need to modify it, we will need to remove it from xml and use in code anyway